tracer/build/_build/UpdateVendors/VendoredDependency.cs

```diff
@@ -104,7 +104,7 @@ private static void RewriteCsFileWithStandardTransform(string filePath, string o
                     {
                         // Disable analyzer
                         var builder = new StringBuilder(AutoGeneratedMessage, content.Length * 2);
-
+                        builder.AppendLine("#pragma warning disable CS0618, CS0649, CS1574, CS1580, CS1581, CS1584, SYSLIB0011,SYSLIB0032");
```

Comment:
nit: might be nice to add a comment here about what these warnings we're disabling are? 🙂

Reply:
Done in 0f91614990db6c6c60bf297f8fc88edce4841621